packages/jitar-reflection/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "jitar-reflection",
+  "version": "0.1.0",
+  "description": "JavaScript reflection library for the Jitar runtime.",
```

Comment:
As long as we don't want to publish this, we should add the "private": true to prevent accidental release.